dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_22__Potential_Duplicate_Not_Null_teiB.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE potentialduplicate ALTER COLUMN teiB SET NOT NULL;
```

Comment:
What if there are already null values in the DB?
Should we delete the invalid rows before applying this script?

Reply:
good point, yes i think we should also because i don't think it makes sense to have missing element in a pair of duplicates @stian-sandvold

Reply:
It is probably not a problem but `teiB` should actually be `teib` as in the first statement